src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -204,6 +204,30 @@ static Activity watching(@Nonnull String name)
         return EntityBuilder.createActivity(name, null, ActivityType.WATCHING);
     }
 
+    /**
+     * Creates a new Activity instance with the specified name.
+     * <br>This will display as {@code Competing in name} in the official client
+     * 
+     * @param  name
+     *         The not-null name of the newly created game
+     * 
+     * @throws IllegalArgumentException
+     *         If the specified name is null, empty, blank or longer than 128 characters
+     * 
+     * @return A valid Activity instance with the provided name with {@link net.dv8tion.jda.api.entities.Activity.ActivityType#COMPETING}
+     *
+     * @incubating This feature is not yet confirmed for the official bot API
```

Comment:
Are you sure about that? Seems to be documented and working

Reply:
I essentially took this from the `watching` method which has pretty much the same description and also mentions that it isn't confirmed for the official api, despite it having the same amount of documentation as the compete one does.

Reply:
https://discord.com/developers/docs/topics/gateway#activity-object-activity-types doesn't mention watching.

Reply:
Must have overlooked it then. Will remove that then.